matlab_simulation/04-sensor/scanner_bresenham/create_a_map.m

```diff
@@ -0,0 +1,24 @@
+function [ map ] = create_a_map( M, N )
```

Comment:
Great function! maybe it should be placed in "matlab_simulation/10-environments" for use with other examples. The issue I worked on could certainly make good use of it.